main/main.cpp

```diff
@@ -1727,7 +1735,22 @@ Error Main::setup2(Thread::ID p_main_tid_override) {
 
 		// rendering_driver now held in static global String in main and initialized in setup()
 		Error err;
-		display_server = DisplayServer::create(display_driver_idx, rendering_driver, window_mode, window_vsync_mode, window_flags, window_size, err);
+
+		/*
+		 * In the case of editor mode, we use the base color defined by Theme.
+		 * Because the user defined splash and the current editor's theme base color may be too high contrast,
+		 * this may cause the background color to flash when any window within the editor is initialized, especially in slow or debug mode.
+		 */
+		// Godot Default background color.
+		Color init_bg_color = Color(0.21f, 0.24f, 0.29f);
```

Comment:
I'm not too fond of hardcoding this, this is bound to get out of sync if we ever change the default background color.